src/cli/init/git.ts

```diff
@@ -2,9 +2,62 @@ import path from 'path';
 
 import fs from 'fs-extra';
 
-import { createExec } from '../../utils/exec';
+import { Exec, createExec } from '../../utils/exec';
 import { log } from '../../utils/logging';
 
+interface GitHubProject {
+  orgName: string;
+  repoName: string;
+}
+
+export const initialiseRepo = async (
+  exec: Exec,
+  { orgName, repoName }: GitHubProject,
+) => {
+  await exec('git', 'init');
+
+  await exec(
+    'git',
+    '-c',
+    'user.email=<>',
+    '-c',
+    'user.name=skuba',
+    'commit',
+    '--allow-empty',
+    '--author',
+    'skuba <>',
```

Comment:
This is hyper-defensive so we don't blow up if you haven't configured your Git username or email 🤷